src/app/component/calendar/appointment-screen/appointment-screen.component.ts

```diff
@@ -27,7 +27,7 @@ export class MyModule {}
 
 export class AppointmentScreenComponent implements OnInit {
 
-  moduleControl = new FormControl(null, [Validators.required, this.isModule]);
```

Comment:
Der isModule Validator sollte eher so angepasst werden dass er auch leere Module zulässt. Komplett ohne Validation wäre es jetzt auch möglich nur einen String in Den ModulePicker zu schreiben und das lässt er durch.

Also: Entweder modul oder leer

Reply:
Diese Form des Validators sollte auch im FlashCardBundleChangeScreen zu finden sein.